applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -568,46 +618,58 @@ static void bt_audio_codec_allocation_set(struct bt_audio_codec_cfg *codec_cfg,
 	}
 }
 
-static int update_sink_stream_qos(struct le_audio_headset *headset, uint32_t pres_delay_us)
+static int update_cap_sink_stream_qos(struct le_audio_unicast_server *unicast_server,
+				      uint32_t pres_delay_us)
 {
 	int ret;
 
-	if (headset->sink_stream.ep == NULL) {
+	if (unicast_server->cap_sink_stream.bap_stream.ep == NULL) {
 		return -ESRCH;
 	}
 
-	if (headset->sink_stream.qos == NULL) {
-		LOG_WRN("No QoS found for %p", &headset->sink_stream);
+	if (unicast_server->cap_sink_stream.bap_stream.qos == NULL) {
+		LOG_WRN("No QoS found for %p", (void *)&unicast_server->cap_sink_stream.bap_stream);
 		return -ENXIO;
 	}
 
-	if (headset->sink_stream.qos->pd != pres_delay_us) {
+	if (unicast_server->cap_sink_stream.bap_stream.qos->pd != pres_delay_us) {
+		struct bt_cap_unicast_audio_stop_param param;
+		struct bt_cap_stream *streams[2];
```

Comment:
Just did a quick go through again, maybe this should be `ARRAY_SIZE(unicast_servers)`?

Reply:
No, this is just for reconfiguring an already started device, so it will only ever start the number of streams that device has. So for now it can max be 2 (sink + source)